From 1b9d2b5078160c129279e37d9121fd60d2e936f4 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Tue, 29 Jan 2019 01:58:48 +0000
Subject: [PATCH 770/909] arm64: dts: r8a7796-m3ulcb: Select the device tree of
 R-Car M3 ES1.x.

commit f8c354beddcd0fb39fd28d7ee1efbf10b12c6772 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

The device tree of R-Car M3 was divided into ES1.x and ES3.0.
M3ULCB is equipped with ES1.0 SoC.
So, this patch selects the device tree of ES1.x.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
index c962301b5f34..ddb451b84448 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7796-m3ulcb.dts
@@ -2,16 +2,16 @@
 /*
  * Device Tree Source for the M3ULCB (R-Car Starter Kit Pro) board
  *
- * Copyright (C) 2016-2018 Renesas Electronics Corp.
+ * Copyright (C) 2016-2019 Renesas Electronics Corp.
  * Copyright (C) 2016 Cogent Embedded, Inc.
  */
 
 /dts-v1/;
-#include "r8a7796.dtsi"
+#include "r8a7796-es1.dtsi"
 #include "ulcb.dtsi"
 
 / {
-	model = "Renesas M3ULCB board based on r8a7796";
+	model = "Renesas M3ULCB board based on r8a7796 ES1.0";
 	compatible = "renesas,m3ulcb", "renesas,r8a7796";
 
 	cpus {
-- 
2.17.1

